### ISSUE TITLE
response sample duplication

### ISSUE THREAD

**Issue by mrtristan** [Mermade/widdershins#134]
commented here as well but figured i should open a new ticket: https://github.com/Mermade/widdershins/issues/129

i'm seeing response body duplication. it seems that i've isolated what's happening down to the widdershins layer in the stack.

please let me know if there is any further information i can provide

result:
![image](https://user-images.githubusercontent.com/1546181/41330587-5eda81a4-6ea2-11e8-801a-b9bf4ae3b9d3.png)

swagger json:
![image](https://user-images.githubusercontent.com/1546181/41330593-6f1b3626-6ea2-11e8-9882-701b2feaff84.png)

markdown:
![image](https://user-images.githubusercontent.com/1546181/41330606-7cb82fc8-6ea2-11e8-8b08-96b80b8a2354.png)

**Comment by MikeRalphson** [comment]
Apologies, I'm really busy this week. Would appreciate any eyeballs taking a look at recent commits (esp. changes to templates) which may have caused this.

**Comment by mrtristan** [comment]
my first thought would be here https://github.com/Mermade/widdershins/commit/a08df5f4c1c2968f51d06b3bb782e84818f19cfc

i'll look in to it if i wind up with a chance, but not looking good for me either this week.

**Comment by MikeRalphson** [comment]
Could you provide a sample OAS document in a gist rather than as a screenshot? That would help me enormously. Thanks.

**Comment by mrtristan** [comment]
apologies, here's a sample:

```
{"swagger":"2.0","info":{"version":"1.0","title":"Api 1.0"},"host":"xxx.com","paths":{"/api/v1/Orders/{id}":{"get":{"tags":["Orders"],"summary":"Get a particular order","description":"Order get remarks","operationId":"ApiV1OrdersByIdGet","consumes":[],"produces":["text/plain","application/json","text/json"],"parameters":[{"name":"id","in":"path","required":true,"type":"integer","format":"int32"}],"responses":{"200":{"description":"Order successfully retrieved","schema":{"$ref":"#/definitions/OrderResponse"}},"401":{"description":"Requestor does not have access to the requested order"},"404":{"description":"Not Found"},"500":{"description":"Error processing"}}},"delete":{"tags":["Orders"],"summary":"Cancels a particular order","description":"Order Cancellation remarks","operationId":"ApiV1OrdersByIdDelete","consumes":[],"produces":[],"parameters":[{"name":"id","in":"path","required":true,"type":"integer","format":"int32"}],"responses":{"200":{"description":"Cancellation succeeded"},"401":{"description":"Requestor does not have authority to cancel order"},"404":{"description":"Order not found for provided id"},"500":{"description":"Error processing"}}}},"/api/v1/Orders":{"post":{"tags":["Orders"],"summary":"Create an order","description":"Order creation remarks","operationId":"ApiV1OrdersPost","consumes":["application/json-patch+json","application/json","text/json","application/*+json"],"produces":["text/plain","application/json","text/json"],"parameters":[{"name":"order","in":"body","required":true,"schema":{"$ref":"#/definitions/OrderRequest"}}],"responses":{"201":{"description":"Order successfully created","schema":{"$ref":"#/definitions/OrderResponse"}},"400":{"description":"Error in the request"},"500":{"description":"Error processing"}}}}},"definitions":{"OrderResponse":{"type":"object","properties":{"id":{"type":"string"},"lineItems":{"type":"array","items":{"$ref":"#/definitions/OrderResponseLineItem"}}}},"OrderResponseLineItem":{"type":"object","properties":{"id":{"type":"string"},"type":{"$ref":"#/definitions/OrderLineItemType"}}},"OrderLineItemType":{"enum":["Delivery","Link","Code"],"type":"string"},"OrderRequest":{"type":"object","properties":{"lineItems":{"type":"array","items":{"$ref":"#/definitions/OrderRequestLineItem"}}}},"OrderRequestLineItem":{"type":"object","properties":{"type":{"$ref":"#/definitions/OrderLineItemType"}}}}}```

**Comment by MikeRalphson** [comment]
Ok, issue was more than one `produces` content-type which mapped (in this case) to auto-generated `json` examples. I've amended this so only one auto-generated example should be produced at maximum for each of `json`, `yaml` and `xml`.

Just waiting on Travis CI and I'll publish a new version.

**Comment by MikeRalphson** [comment]
Should be fixed in `v3.4.3`. Please chase if not.
